conda_smithy/linter/hints.py

```diff
@@ -247,9 +247,9 @@ def hint_noarch_python_use_python_min(
     if noarch_value == "python" and not outputs_section:
         hint = ""
         for section_name, syntax, reqs in [
-            ("host", "python {{ python_min }}", host_reqs),
-            ("run", "python >={{ python_min }}", run_reqs),
-            ("test.requires", "python {{ python_min }}", test_reqs),
+            ("host", r"python\s+{{ python_min }}", host_reqs),
+            ("run", r"python\s+>={{ python_min }}", run_reqs),
+            ("test.requires", r"python\s+{{ python_min }}", test_reqs),
```

Comment:
```suggestion
            ("host", r"python\s+\{\{ python_min \}\}", host_reqs),
            ("run", r"python\s+>=\{\{ python_min \}\}", run_reqs),
            ("test.requires", r"python\s+\{\{ python_min \}\}", test_reqs),
```

Reply:
We might also need `\s*` because the spaces there are technically optional, IIRC

Reply:
I don't think the spaces are optional. Also we long on that in another spot.